charts/simple/templates/deployment.yaml

```diff
@@ -101,7 +101,7 @@ spec:
         resources:
           {{- .Values.signalsciences.resources | toYaml | nindent 10 }}
       {{- end }}
-
+      serviceAccountName: {{ include "simple.serviceAccountName" . }}
```

Comment:
Why does this not replace line 134 (imagepullsecrets) like it does in the drupal-project-k8s PR?

Reply:
fixed now